spec/genernal-entity/general.entity.ts

```diff
@@ -0,0 +1,39 @@
+import { ApiHideProperty, ApiProperty } from '@nestjs/swagger';
```

Comment:
There is typo in directory name.
- `spec/genernal-entity/general.entity.ts` -> `spec/general-entity/general.entity.ts`

Reply:
thanks